website/docs/pchoutputfile.md

```diff
@@ -0,0 +1,28 @@
+Specifies the output file generated by the precompiled header
+
+```lua
+pchoutputfile ("filename.pch")
+```
+
+See [Precompiled Headers](Precompiled-Headers.md) for more information.
+
+### Parameters ###
+
+`filename.pch` is the name of the outputted precompiled header file.
+
+On MSVC this is the `/FP` compiler flag.
+
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
+
+Premake 5.0 and up.
```

Comment:
Not sure it is the correct version.

Reply:
I assumed 5.0 is still in beta and wasn't "released yet" but once its released it would have this feature, I'm honestly not sure either.

Reply:
I hope its the correct way to change msc.lua

Reply:
It would be 5.0.0 beta 3 or later